anime-list-master.xml

```diff
@@ -28562,8 +28563,8 @@
   <anime anidbid="10384" tvdbid="79214" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Mushishi Zoku Shou</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0">;1-2;2-3;</mapping>
-      <mapping anidbseason="1" tvdbseason="0">;11-4+5;</mapping>
+      <mapping anidbseason="0" tvdbseason="0">;1-0;2-0;</mapping>
```

Comment:
No need to unmap AniDB Specials if it's not the only season for a TVDB ID

Reply:
```xml
<mapping anidbseason="0" tvdbseason="0">;1-2;2-3;</mapping>
```

The current mapping for specials is wrong. Specials for this `AniDB` entry have no relevant `TVDB` episodes.

[S1](https://anidb.net/episode/160019) | Mushistory
-- | -- 
[S2](https://anidb.net/episode/160697) | Mushistory 2

Additionally, media scanners (such as [Absolute Series Scanner](https://github.com/ZeroQI/Absolute-Series-Scanner)) wouldn't handle metadata properly because there are multiple `AniDB` entries with specials for the same `TVDB` entry.